chroma-agent/Target

```diff
@@ -53,44 +53,57 @@ The name of the target.
 END
 }
 
+dbg() {
+    echo "`date` RA $1" >> /var/log/chroma-agent.log
+}
+
 Target_start() {
+    dbg "Target_start()"
     # See if the device is already mounted.
     if Target_status >/dev/null 2>&1; then
-        ocf_log info "Target $TARGET is already started."
+        # ocf_log info "Target $TARGET is already started."
```

Comment:
to be honest is because I could not work out why / where the logs were ending up, we've spent a number of years with these log entries not going anywhere where we can see them, this change at least means they end up somewhere useful where we can see them. in another patch we should investigate and fix ocf_log.